database/migrations/2022_08_18_133641_add_removed_properties_to_forms.php

```diff
@@ -14,7 +14,7 @@
     public function up()
     {
         Schema::table('forms', function (Blueprint $table) {
-            $table->jsonb('removed_properties')->default('[]');
+            $table->text('removed_properties');
```

Comment:
Removing defaults breaks the application (see the tests in the workflow run). If you can't have them in the database layer, you at least need them at the application layer (Maybe in `$attributes` on the model?)

Reply:
Why not json filed also? `text` is not the right type here

Reply:
I don't think that json or jsonb work in mysql. Hence, we need to parse it as text

Reply:
resolved